skabase/SKATestDevice/SKATestDevice/SKATestDevice.py

```diff
@@ -199,55 +199,55 @@ def RunGroupCommand(self, argin):
         # PROTECTED REGION END #    //  SKATestDevice.RunGroupCommand
 
     def dev_logging(self, dev_log_msg, dev_log_level):
-        #Element Level Logging
-        if self._element_logging_level >= int(tango.LogLevel.LOG_FATAL) \
-                and dev_log_level == int(tango.LogLevel.LOG_FATAL):
+        # Element Level Logging
+        if self._element_logging_level >= int(tango.LogLevel.LOG_FATAL) and dev_log_level == int(
+                tango.LogLevel.LOG_FATAL):
```

Comment:
OK, although using parenthesis around it all, and having one term per line, would be more readable for me.
```python
if (self._element_logging_level >= int(tango.LogLevel.LOG_FATAL) and
        dev_log_level == int(tango.LogLevel.LOG_FATAL)):
    self.fatal_stream(dev_log_msg)
```
As suggested in pep8, the second term has extra indentation so that it doesn't line up with the code in the body of the if statement.

By the way, the casting to int for the enumerated type isn't necessary either.  E.g.
```python
In [1]: import tango

In [2]: int(tango.LogLevel.LOG_WARN)
Out[2]: 3

In [3]: 1 > tango.LogLevel.LOG_WARN
Out[3]: False

In [4]: 4 > tango.LogLevel.LOG_WARN
Out[4]: True

In [5]: 4 == tango.LogLevel.LOG_WARN
Out[5]: False

In [6]: 3 == tango.LogLevel.LOG_WARN
Out[6]: True
```